flow/project.py

```diff
@@ -993,10 +1168,10 @@ def _create_run_job_operations(self, entrypoint, default_directives, job,
             the argument must be explicitly set to an empty dictionary.
         :type default_directives:
             dict
-        :param job:
-            The job that the :class:`~._JobOperation` is based on.
-        :type job:
-            :class:`signac.Job`
+        :param jobs:
+            The list of jobs that the :class:`~._JobOperation` is based on.
+        :type jobs:
+            list of :class:`signac.Job`
```

Comment:
`signac.Job` is not the correct namespace of the class. It should be `signac.contrib.job.Job`. I think this is a mistake that has spread through the docstrings, so you may have copied it from somewhere else that was also wrong. Please fix this everywhere it occurs in this PR (this is the first case I've caught).
```suggestion
            list of :class:`signac.contrib.job.Job`
```